plugins/Debug_ReportMemory.js

```diff
@@ -1,3 +1,19 @@
+/*:
+ * @plugindesc Show memory usage.
+ * @author RM CoreScript team
+ *
+ * @help
+ * This plugin shows memory related usages.
+ * size: size in MPixel
+ * totalCount: count of Bitmaps
+ * requestCount: count of requested Bitmaps
+ * reservedCount: count of reserved Bitmaps
+ *
+ * All number should be stable.
+ * If you find that It increases gradually, Please report to team!!
```

Comment:
I think that should be written accurately here.
The number will increase for the beginning.